supporting_scripts/format_slow_query_report.py

```
@@ -0,0 +1,158 @@
+            issues.append(f"{slow_count} slow quer{'y' if slow_count == 1 else 'ies'}")
+        if n1_count > 0:
+            issues.append(f"{n1_count} N+1 suspect{'s' if n1_count != 1 else ''}")
+        header_status = "Performance issues found: " + " · ".join(issues)
+
+    lines = [
+        "",
+        "---",
+        f"## {header_emoji} Slow Query Report",
+        "",
+        f"> Generated at: {generated_at}  ",
+        f"> Slow-query threshold: **{threshold_ms} ms** · N+1 detection: **>{n1_threshold}×/request**",
```

Ruff (0.15.20)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)